django/contrib/sessions/backends/base_cached_db.py

```diff
@@ -0,0 +1,85 @@
+"""
```

Comment:
Is the need for a separate file due to the fact that `cached_db.py` imports the `Session` model? Could we fix that be moving the import into the `get_session_class()` method instead of having it at the module level?